test.Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM quay.io/ebi-ait/ingest-base-images:python_3.7-slim
```

Comment:
Where's this Dockerfile used?

Reply:
The readme needs a massive overhaul, all of this information is out of date and has been for a long time. But I've updated the section about unittests.

Reply:
Thanks!